code/src/components/TaskCounter.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+/* Counter component, it shows how many todos i completed, and how many are left.
+we filter the tasks depening on if they are completed or not */
+
+const TaskCounter = () => {
+  const allTasks = useSelector((store) => store.tasks.items)
+
+  const completedTasks = allTasks.filter((singleTask) => singleTask.complete)
+  const tasksLeft = allTasks.filter((singleTask) => !singleTask.complete)
+
+  return (
+    <CompletedToDos>
+      <StyledParagraph>Completed todos: {completedTasks.length} / {allTasks.length}
+      </StyledParagraph>
+      <StyledParagraph>Todos left: {tasksLeft.length}
+      </StyledParagraph>
+    </CompletedToDos>
+  )
+}
+
+/* styling components */
+
+const CompletedToDos = styled.article`
```

Comment:
Maybe you could make a reusable post-it styling ing Global style with props so you don't have to redo it all the time?